prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -1198,5 +1198,27 @@ public async Task<ActionResult> FlagSite(int siteId, FromBodyData<bool> flagged)
             await _communitySiteService.UpdateSiteFlag(siteId, flagged);
             return Ok(site);
         }
+
+        // GET: api/sites/5/individual-device-providers
+        /// <summary>
+        /// Gets the Individual Device Providers for a Device Provider Site.
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpGet("{siteId}/individual-device-providers", Name = nameof(GetIndividualDeviceProviders))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<IndividualDeviceProviderViewModel>>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> GetIndividualDeviceProviders(int siteId)
```

Comment:
If this is an endpoint that can be hit by the user, we need to check the permissions record. If its an admin only endpoint, this needs `[Authorize(Roles = Roles.ViewSite)]`